README.md

```diff
@@ -18,7 +18,10 @@ $ npm install express-session
 ## API
 
 ```js
-var session = require('express-session')
```

Comment:
This line yielded the following error.

```
.../express-session/README.md
  21:5  error  'session' is assigned a value but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)
```

Instead of skipping this, I though it would be ok to expand this introduction usage a bit. What do you think?